### DISCUSSION TITLE
I'm totally stuck

### DISCUSSION THREAD

**Discussion by lennielynn** [vercel/next.js#47410] [Help]
### Summary

how do i open pages/index.js in my text editor. I am using VS code but cannot figure out how to open that page. So, Help!!

**Comment by icyJoseph** [comment]
You should be able to open the project from VSCode's UI:

- https://learn.microsoft.com/en-us/visualstudio/get-started/tutorial-open-project-from-repo?view=vs-2022

And then the pages file should be available there.
